lms/templates/problem_ajax.html

```diff
@@ -1 +1 @@
-<div id="problem_${element_id}" class="problems-wrapper" data-problem-id="${id}" data-url="${ajax_url}" data-progress_status="${progress_status}" data-progress_detail="${progress_detail}"></div>
+<div id="problem_${element_id}" class="problems-wrapper" data-problem-id="${id}" data-url="${ajax_url}" data-progress_status="${progress_status}" data-progress_detail="${progress_detail}" data-content="${content | h}"></div>
```

Comment:
Okay, I'll give it a shot -- will probably follow up with an actual story for the next sprint though. I did this mostly as a proof of concept alternative while doing the XBlock handler CDN spike.